src/vs/workbench/electron-browser/workbench.ts

```diff
@@ -74,6 +74,11 @@ import {IEnvironmentService} from 'vs/platform/environment/common/environment';
 import {ITelemetryService} from 'vs/platform/telemetry/common/telemetry';
 import * as watermark from 'vs/workbench/parts/watermark/watermark';
 
+// These three are used for the macOS inline toolbars option, and can be removed once default
+import { IConfigurationService } from 'vs/platform/configuration/common/configuration';
+import { ConfigurationService } from 'vs/platform/configuration/node/configurationService';
```

Comment:
Can you make sure the formatting of imports matches the other imports? also I suggest to remove the comment for now, we will likely not make this default soon nor get rid of the setting if people don't like it. We still have to figure out how to run this experiment, maybe settings is not the best in the end.